Makefile

```diff
@@ -63,7 +63,7 @@ test-dependencies:
 	@pip install -q -r test-requirements.txt
 
 test: test-dependencies test-start-minio ## run tests quickly with the default Python
-	coverage run -m pytest -v || $$(make test-stop-minio)
+	- coverage run -m pytest -v
```

Comment:
Isn't ignoring the result here going to lead to CI always being successful?  Actually, it would always reflect the result of the `test-stop-minio` target.  We need the `make test` result to reflect the result of `pytest -v`.

I found the following to work (but you should probably check as well):
```
test: test-dependencies test-start-minio ## run tests quickly with the default Python
        coverage run -m pytest -v etc/tests/test || r=$$?; $$(make test-stop-minio); exit $$r
        @make test-stop-minio

test-stop-minio: ## stop test_minio container (dev testing)
        @-docker rm -f test_minio > /dev/null 2>&1
```

Rather than wait for the suite to finish, I found two tests (one successful and one failure) and updated the `pytest -v` portion to read:
`pytest -v etc/tests/test_bootstrapper.py::test_main_method_with_dir_outputs` (to test failures, since this was failing for me) and
`pytest -v etc/tests/test_bootstrapper.py::test_fail_bad_endpoint_main_method` (to ensure the success case worked)

then echo'd `$?` to ensure the command's proper exit status.

Reply:
> Isn't ignoring the result here going to lead to CI always being successful?

You are right, but that's been the behavior before the change as well and I wasn't sure if that was intentional. That said, I did tune your suggestion to remove the duplication. See next update!

Reply:
I should also add that I didn't realize what was really causing the error. For the record, `docker rm -f test_minio` writes output to the STDOUT stream, which `$$(make...` was trying to execute.